app/views/admin/shared/_sidenav_users.html.haml

```diff
@@ -1,5 +1,5 @@
 %ul.sidebar-nav
-  - if current_ability.can?(:administer, User)
+  - if current_ability.can?(:manage, User)
```

Comment:
This has some interesting behavior. If I'm a director, when I go to the users's tab, I this this item highlighted. But as soon as I go to either "bulk email" or "payment sources", this item disappears.